src/genie/libs/parser/hp_comware/tests/DisplayInterfacesBrief/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,75 @@
+expected_output = {
+    "bridge": {
```

Comment:
@tahigash, @ahersi2-cisco 
What about change this schema to:
```python
"interfaces" : {
  "Bridge-Aggregation1": {
    "if_type": "bridge",
   ...
  }
  "Vlan1" : {
    "if_type":  "route",
    ....
    }
}  
```
That way the schema keeps similar to the others.

Reply:
actually, that also works as long as the interface exists only in either one `briege` or `route`. I'm not entirely sure about the output on Comware. but personally I prefer to see similar to CLI output like current schema.

And if there is situation we want to pickup only for `bridge` or `route`, easy with current schema by accessing to `parsed_output['bridge'].keys()`.

Reply:
@tahigash this is the case, the comware output shows `bridge` interfaces separate from `route`, but if you want to deploy the genie in a multi-vendor environment, it is useful to have similar schemas, even if it means having the schema distinct from the output.
Don't know which one is better what do you think?

Reply:
@renatoalmeidaoliveira if we can have similar schema with other OSes, it would be great. but sometime it's difficult because of differences between OSes. in pyATS/Genie, parser is not only way to do something. we can absorb difference even via Genie Ops or API.

For this parser, I don't see any issue with either schema. so if you are planning to use this in multi-vendor environment, please update the schema to later one.

Reply:
@tahigash I think the best is to keep this way, because these schema is similar to the device output.